packages/user-group-service/src/groups/typedef.graphql

```diff
@@ -0,0 +1,51 @@
+type Query {
+  """
+  Returns a list of all Groups (default limit=25)
+  """
+  listGroups(limit: Int): [Group]
+  """
+  Returns a list of Groups for the given selector query (default limit=25)
+  """
+  getGroupsBy(selector: GetGroupInput!, limit: Int): [Group]
+}
+
+type Mutation {
+  """
+  Add a new Group Mapping (i.e. A new One Platform Group corresponding to a Rover Group)
+  """
+  addGroup(payload: AddGroupInput!): Group
+  """
+  Updates a Group with id using values from the payload
+  """
+  updateGroup(id: ID!, payload: UpdateGroupInput!): Group
+  """
+  Removes a group matching the given inputs
+  """
+  deleteGroup(id: ID!): Group
+}
+
+scalar DateTime
+
+type Group {
+  _id: ID
+  name: String
+  ldapCommonName: String
```

Comment:
I made it `ldapCommonName` to avoid any ambiguity for users when using it.

But if needed, can change it back to `cn`.

@diwanshi @sayak-sarkar @riginoommen which should we keep?

Reply:
I would ideally prefer `cn` since it would help to keep things standardized.

Reply:
If we want to standardize, then IMO it should be ldapCommonName as that's the one used in Rover. And we're going to use Rover APIs.